### PR TITLE
Completed javascript-primitives-and-variables exercise

### PR DESCRIPTION
**Pull request by jessicagrayson**
javascript-primitives-and-variables - Jessica Grayson
Screen Shot 2023-07-10 at 1 52 01 PM

### CODE REVIEW

File: javascript-primitives-and-variables/javascript-primitives-and-variables-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+- What does the `=` operator mean in JavaScript?
+
+- How do you update the value of a variable?
+
+- What is the difference between `null` and `undefined`?
+
+- Why is it a good habit to include "labels" when you log values to the browser console?
+
+- Give five examples of JavaScript primitives.
```
Comment (shawnkost) [inline javascript-primitives-and-variables/javascript-primitives-and-variables-quiz-notes.md:31]: Don't forget the quiz questions! These need to be answered & submitted along with your code.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Code looks good Jessica! Just need to answer the quiz questions

File: javascript-primitives-and-variables/javascript-primitives-and-variables-quiz-notes.md (hunk 1 above)
Comment (jessicagrayson) [reply javascript-primitives-and-variables/javascript-primitives-and-variables-quiz-notes.md]: Quiz questions should now be updated. Sorry about that!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**